index.js

```diff
@@ -17,11 +17,12 @@
 
 import { APMTracer } from './lib/APMTracer';
```

Comment:
Minor point, is there a convention regarding the lib naming (case wise)? The recorder and sampler are lowercase, but APM uses capitals.